### PR TITLE
Issue 10: Add the basic docs setup

### PR DESCRIPTION
**Pull request by oscarpolanco**
Adding basic docs structure to the `clay` repo.

Issue: https://github.com/clay/clay/issues/10

# Steps for testing the site locally:
- Go to the `website` folder
- Install all dependencies using `npm install`
- Start your local server using `npm run start`
- Test locally at [localhost](http://localhost:3000/)

# Steps to test the build:
- Go to the `website` folder
- Install all dependencies using `npm install`
- Start your local server using `npm run build`
- Check that now you got a `build` folder under the `website` directory

# Notes: 
- The deploy process is ready but is no automatic and we still need to add the repo that will have all the build files that `Github` pages need.
- We gonna do the first build to `gh-pages` after we merge this branch
- Leave some of the default files for future references

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "publish-gh-pages": "docusaurus-publish",
+    "write-translations": "docusaurus-write-translations",
+    "version": "docusaurus-version",
+    "rename-version": "docusaurus-rename-version"
+  },
+  "devDependencies": {
+    "docusaurus": "^1.7.2"
+  },
+  "dependencies": {
+    "gh-pages": "^2.0.1"
```
Comment (oscarpolanco) [inline website/package.json:15]: After this [PR](https://github.com/clay/claycli/pull/135) is merge we will add the `action` for the deploy

File: website/pages/en/index.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+          <Container padding={['right', 'left']}>
+            <MarkdownBlock>
+              Clay is an open-source CMS created by [New York Media](http://mediakit.nymag.com/) powering [New York Magazine](http://nymag.com/), 
+              [Vulture](https://www.vulture.com/), [The Cut](https://www.thecut.com/), [Grub Street](http://www.grubstreet.com/) and [Slate](https://slate.com/). 
+
+              Clay is comprised of modules available on npm, but the core system is comprised of [Kiln](https://claycms.gitbook.io/kiln), the editing
+              interface, and [Amphora](https://claycms.gitbook.io/amphora), the REST API. For more information on the two modules please see
+              documentation on each. The documentation in this space is meant to explain core concepts of Clay, but
+              for a deeper technical dive you can explore the documentation for each of the modules themselves.
+            </MarkdownBlock>
```
Comment (oscarpolanco) [inline website/pages/en/index.js]: I can't pull content directly from the docs on the `index.js` file 😅

File: website/static/css/custom.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+@media only screen and (max-width: 1023px) {
+}
+
+@media only screen and (min-width: 1400px) {
+}
+
+@media only screen and (min-width: 1500px) {
+}
```
Comment (oscarpolanco) [inline website/static/css/custom.css:16]: Leave this file in case we need it when we got the final design

File: website/pages/en/index.js [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+            <div className="mainContent">
+              <MarkdownBlock>
+                Clay is comprised of modules available on npm, but the core system is comprised of [Kiln](https://claycms.gitbook.io/kiln); the editing
+                interface; and [Amphora](https://claycms.gitbook.io/amphora); the REST API.
+              </MarkdownBlock>
+            </div>
+            <div className="mainContent">
+              <MarkdownBlock>
+                The documentation in this space is meant to explain core concepts of Clay,
+                for a deeper technical dive, you can explore the documentation of each of the modules themselves.
```
Comment (oscarpolanco) [inline website/pages/en/index.js:118]: Need to check the repos that are public to put a list of them

**Review by CharlieGarcia — APPROVED** [review]
CLGTM

**Review by jonwinton — APPROVED** [review]

**Pull request merged**